x-pack/plugins/fleet/common/openapi/bundled.json

```diff
@@ -1316,7 +1323,9 @@
                     "online",
                     "other",
                     "total",
-                    "updating"
+                    "updating",
+                    "all",
+                    "active"
```

Comment:
Not sure if the `required` list is supposed to be alphabetically sorted. Looks like it was before this change. 🤔

Reply:
I don't think it matters (sorry the pr was on auto merge)